docs/events.md

```diff
@@ -0,0 +1,179 @@
+# Adding Events
+
+You can add an event or training to the site by adding a markdown file in the [_events](_events)
+folder, organized by year. 
+
+Events are displayed at https://us-rse.org/events
+
+You can use the [add an event form](https://docs.google.com/forms/d/e/1FAIpQLSda4-gAKyVA1GhJZg3XZmc9EDLaf5Donlm1HKG6r8ve9ooiRQ/viewform) instead if you don't know how to submit a PR. 
+
+## Naming the File
+
+Do not use the full date (e.g. YYYY-MM-DD-<event-name>.md) in the file name,
+Jekyll will not post pages that it interprets to have a future date in the filename. Use a partial date (e.g. YYYY-MM-<event-name>.md or YYYY-<event-name>.md). 
+
+## Header information
+
+Here is an example of a file in `_events/2019` for PEARC19:
+
+```markdown
+---
+title: PEARC19
+location: Chicago, IL
+expires: 2019-08-01
+duration: 45
+category: workshop
+event_date: "July 29, 2019"
+layout: event
+repeated: false
+time:
+    - - start: 2019-07-29T21:15:00Z
+        end: 2019-07-29T22:00:00Z
+---
+
+Join us at [PEARC19](https://www.pearc19.pearc.org/) for a Birds of a Feather (BOF) session "Building a Community of Research Software Engineers."  Our session is scheduled for 5:15 PM on Monday, July 29.
+```
+
+* title: Keep it brief to display at the top of the event card
+* location: optional, can leave blank for online events
+* expires: usually the day of the event, YYY-MM-DD
+* duration: time in minutes, used for calendar export; defaults to 60
+* category: see [_data/events.yaml](_data/events.yaml) - use the tag (e.g. dei, virtual-workshop), name is the display name on the page for the tag
+* event_date: display text for the date, specific format not required (not displayed in all contexts, so don't rely on it)
+* layout: "event"
+* repeated: true or false
+* time: the format is very specific; times should be in UTC
+
+
+## Content
+
+The bottom section (the content) you can write any amount and length
+of markdown that is desired. When the event is active (before expiration) the full content will
+be shown on the "Events and Training" page. Once it expires, it will move into the events archive.
+In both cases, clicking on the Event will take the viewer to its page, and they can
+view additional content and the url provided. In the case of the archive, the bulk of content
+is only viewable on this page.
+
+It's helpful to write out a listing of timezones in the content, e.g.,:
+
+```md
+The next community call will be on August 12, 2021 at 12ET/11CT/10MT/9PT.
```

Comment:
We should figure out what format we want to use for these. I personally don't like these without a space between the number and the timezone...  But maybe this is what others decided we are doing to use?

Reply:
I just grabbed a recent example - happy to put in a different format if there are suggestions

Reply:
I don't think we've had any agreement on this, but the website group could make the decision

Reply:
I think there should at least be spaces and the time zone abbreviations should be complete, e.g.,  "August 12, 2021 at 12pm EDT/11am CDT/10am MDT/9am PDT".